api/src/Database/Type/PureMySQL.php

```diff
@@ -0,0 +1,262 @@
+<?php
```

Comment:
This file is very similar to MySQL.php, is there a way to extend MySQL.php with what you need?

Reply:
PureMySQL removes functionality from MySQL, so extending it will not be useful.

Reply:
After discussions, will leave as is as it removes oracle functionality which may be useful for new pages/endpoints in future.